applications/nrf_desktop/README.rst

```diff
@@ -1555,52 +1555,61 @@ This code snippet will change the **SPI1** interrupt priority from default ``1``
 
 .. _nrf_desktop_flash_memory_layout:
 
-Flash memory layout
-===================
-
-The flash memory is set by defining one of the following options:
-
-* `Memory layout in DTS`_ (bootloader is not enabled)
-* `Memory layout in partition manager`_ (either bootloader or :ref:`nrf_desktop_bluetooth_guide_fast_pair` is enabled)
+Memory layout
+=============
 
+An application developer must define memory layout as part of the application configuration for a given board.
 The set of required partitions differs depending on configuration:
 
 * There must be at least one partition where the code is stored.
 * There must be one partition for storing :ref:`zephyr:settings_api`.
 * The bootloader, if enabled, will add additional partitions to the set.
+* When using an SoC with multiple cores, the firmware for additional cores will add additional partitions to the set.
+  For example, nRF53 SoC uses ``HCI RPMsg`` firmware image for the network core to use the core for Bluetooth LE communication.
 
 .. important::
    Before updating the firmware, make sure that the data stored in the settings partition is compatible with the new firmware.
    If it is incompatible, erase the settings area before using the new firmware.
 
+The memory layout is defined through one of the following methods:
+
+* `Memory layout in DTS`_
+* `Memory layout in Partition Manager`_
+
+By default, a Zephyr-based application defines memory layout in the DTS.
+If enabled, the :ref:`partition_manager` defines a new memory layout that is used instead of the memory layout defined in the DTS.
+The :kconfig:option:`CONFIG_PARTITION_MANAGER_ENABLED` Kconfig option value can be used to check if the Partition Manager is enabled in the current build.
+The option is automatically selected in case :ref:`ug_multi_image` feature is used to build the application together with a bootloader or nRF53 network core image.
```

Comment:
What is the nRF53 network core image? nRF53 network core has many potential images that you can use. Let's make this sentence more generic and less target specific

```suggestion
The option is automatically selected as part of the :ref:`ug_multi_image` feature to build the application with more than one image.
```

Reply:
Ok, we can update it.